portfolio/withdraw/withdraw_types.go

```diff
@@ -94,6 +113,9 @@ type Request struct {
 	// Used exclusively in Binance.US
 	ClientOrderID string `json:"clientID"`
 
+	// Currently used exclusively in Coinbase
+	WalletID string `json:"walletID"`
```

Comment:
🦍  I was going to move the comment onto the same line as a suffix comment.
But ... this comment isn't adding anything useful.
Neither is the previous one, if you can nuke it at the same time.
```suggestion
	WalletID string `json:"walletID"`
```

🥃  The one about `InternalTransfer` seemed useful. Except that the information is in the wrong place, and doesn't match the type, so the comment belongs somewhere else. But that feels Out-of-Scope.

Reply:
I'll remove the Coinbase and Binance comments.